src/python/clawutil/claw_git_status.py

```diff
@@ -18,7 +18,7 @@
 repos_list = ['classic', 'amrclaw', 'clawutil', 'pyclaw', 'visclaw', 'riemann',
               'geoclaw']
 
-def make_git_status_file(outdir=os.getcwd()):
+def make_git_status_file(outdir='.'):
```

Comment:
What about instead setting `outdir=None` and then in the function calling `os.getcwd()`.  It's probably not too big of a deal but I think this would make this code more portable.